project1/application.py

```diff
@@ -1,24 +1,35 @@
 import os
+import datetime
+import bcrypt
+from models import *
 
 from flask import Flask, session, render_template, request
 from flask_session import Session
 from sqlalchemy import create_engine
 from sqlalchemy.orm import scoped_session, sessionmaker
+import logging
+from flask_sqlalchemy import SQLAlchemy
+from sqlalchemy import exc
+
 
 app = Flask(__name__)
+os.environ["DATABASE_URL"] = "postgres://vdsjxpbuoklrgr:1669a5d62c3c6e5aaf3c580c4245a3703c760912beb63a870d62f93c69e6b45f@ec2-54-88-130-244.compute-1.amazonaws.com:5432/d8ijt27ijeed12"
```

Comment:
Donot store url in code ... anyone who can see the code will get access to your database.

Reply:
ok, removed the url.